### ISSUE TITLE
Framework not found with Xcode 12

### ISSUE THREAD

**Issue by RyanWesley20** [SocketMobile/cocoapods-capture#17]
I'm trying to include the capture SDK From pods but on building the project I'm getting the following error (I was able to build earlier, but having issues with Xcode 12)

Deployment Target: 14.2

Error:

Framework not found SKTCapture

Steps to Reproduce:

1) Create an iOS Project from Xcode (Select App under Application) 
2) Add the following podfile in the project directory:

platform :ios, '14.2'
target 'Test'
  pod 'SKTCapture', '~> 1.2'

3) Run pod install from command line
4) Open Test.xcworkspace and try to build the project

Any assistance would be greatly appreciated as we need to ship out an update for our application ASAP.

Thanks!

**Comment by EricGlaenzer** [comment]
Hi @RyanWesley20,
Sorry to learn you have issue installing the Capture SDK.
This new version (1.2.187) does require the new Cocoapods version: 1.10.0. or higher.
Let us know if this is still not working for you.

**Comment by otusweb** [comment]
I'm running into the same issue and have cocoapod version 1.10.0

**Comment by EricGlaenzer** [comment]
Sorry to learn this @otusweb. What is your Xcode version and the deployment target OS version? (Sometimes removing and adding back the SKTCapture cocoapods could help, I noticed that in a project we have here at Socket. I usually do that by commenting the SKTCapture line in the podfile and then type pod install.

**Comment by otusweb** [comment]
Xcode 12.2
deployment target 11.4
project is a few years old, contains both objectiveC and Swift code

**Comment by EricGlaenzer** [comment]
That's should work then, make sure to get the latest Capture version: 1.2.187 that's the one done using XCFramework format for Xcode 12, and containing both ObjectiveC and Swift should not be a problem either. If you have a build log to share with us, please do so by sending it to sdksupport [at] socketmobile.com. Thank you.

**Comment by chbeer** [comment]
Same here: 
```pod --version
1.10.0```

`The 'Pods-…' target has transitive dependencies that include statically linked binaries: (SKTCapture)`

**Comment by EricGlaenzer** [comment]
Hi @chbeer,
Sorry to read about this.
Did you make sure it’s using the SKTCapture version 1.2.187?

**Comment by chbeer** [comment]
``` % pod install
Analyzing dependencies
Downloading dependencies
Installing SKTCapture (1.2.187)
[!] The 'Pods-…' target has transitive dependencies that include statically linked binaries: (SKTCapture)```

**Comment by otusweb** [comment]
Same here, 
```Analyzing dependencies
Downloading dependencies
Installing SKTCapture 1.2.187 (was 1.2.53)
Generating Pods project```

Podfile
```
# Uncomment this line to define a global platform for your project
minimum_deployment_target = '11.4'
platform :ios, minimum_deployment_target

source 'https://github.com/CocoaPods/Specs.git'
source 'https://bitbucket.org/payworks/io.payworks.repo.pods.git'

use_frameworks!
# ignore all warnings from all pods
inhibit_all_warnings!

def sharedPod
  pod 'Gedatsu', :configurations => ['Debug']
  pod 'IQKeyboardManager'
  pod 'Lokalise', '~> 0.10.0'
#  pod 'LokaliseLiveEdit'
  pod 'MBProgressHUD', '~> 1.2.0'
  pod 'MTBBarcodeScanner'
  pod 'PhoneNumberKit', '~> 3.2'
  pod 'Reveal-SDK', '23',  :configurations => ['Debug']
  pod 'SDWebImage', '5.10.0'
  pod 'SKTCapture', '1.2.187'
  pod 'STTwitter'
  pod 'SearchTextField'
  pod 'SendGrid', :git => 'https://github.com/scottkawai/sendgrid-swift.git'
  pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '6.0.9'
  pod 'SideMenu', '6.5.0'
  pod 'StarIO', '2.8.1'
  pod 'StarIO_Extension', '1.15.0'
  pod 'Stencil', '~> 0.14.0'
  pod 'SwiftFormat/CLI'
  pod 'SwiftGen', '~> 6.0'
  pod 'SwiftLint'
  pod 'TwitterKit5'
  pod 'UICKeyChainStore'
  pod 'YLProgressBar', '~> 3.11.0'
  pod 'payworks', '2.41.0'
  pod 'SwiftySound'
end

target 'xxx' do
sharedPod

end

target 'xxx2' do
sharedPod
end


post_install do |installer|

  # Workaround for AppStore issue
  # ITMS-90381: Too many symbol files
  # https://github.com/CocoaPods/CocoaPods/issues/7111
  installer.generated_projects.each do |project|
    project.targets.each do |target|
      target.build_configurations.each do |config|
        # Ensure we set all Pods to match the minimum deployment target specified by the app
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = minimum_deployment_target
      end
    end
  end

  # fix twitterKit related warnings
  # https://gist.github.com/npu3pak/108d8332a774289c08472d6fd85a1bbb
  installer.aggregate_targets.each do |aggregate_target|
    aggregate_target.xcconfigs.each do |config_name, config_file|
      config_file.other_linker_flags[:frameworks].delete("TwitterCore")

      xcconfig_path = aggregate_target.xcconfig_path(config_name)
      config_file.save_as(xcconfig_path)
    end
  end
end
```


build error
```
ld: framework not found SKTCapture.xcframework
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by EricGlaenzer** [comment]
A new version has been release that should fix your issue: 1.3.3
